packages/cli/src/commands/git/index.ts

```diff
@@ -16,7 +16,7 @@ const help = () => {
 
   ${chalk.dim('Commands:')}
 
-    connect                          Connect your Git config "origin" remote as a Git provider to your project
+    connect [url]                    Connect a local Git remote or given Git provider repository URL as a Git provider to your project
```

Comment:
"local git remote" sounds contradictory

```suggestion
    connect [url]                    Connect your Vercel Project your Git repository or provide the remote URL to your Git repository
```